crates/solidity/outputs/cargo/crate/Cargo.toml

```
@@ -25,16 +25,27 @@ categories = [
   "text-editors",
 ]
 
+[[bin]]
+name = "slang_solidity"
+path = "src/main.rs"
+required-features = ["cli"]
+
+[features]
+default = ["cli"]
+cli = ["anyhow", "clap", "serde_json"]
```

I suggest using the `dep:` syntax, to avoid creating unnecessary features:
https://doc.rust-lang.org/cargo/reference/features.html#optional-dependencies

```suggestion
cli = ["dep:anyhow", "dep:clap", "dep:serde_json"]
```